src/qt/winshutdownmonitor.h

```diff
@@ -10,7 +10,7 @@
 #include <QString>
 #include <functional>
 
-#include <windef.h> // for HWND
+#include <windows.h> // for HWND
```

Comment:
Comment should be removed? We are removing these "for xyz" comments in any case, but this one would also now be incorrect.

Reply:
> Comment should be removed?

Thanks! Done.